src/elements/ProjectCard/projectCard.util.js

```diff
@@ -58,36 +58,36 @@ export const generateProjectStatistic = (
           : isEmpty(projectSummary)
           ? '-'
           : projectSummary.runs_failed_recent_count + failedNuclioFunctions
-    },
-    models: {
-      className: 'default',
-      label: 'Models',
-      loading: projectsSummaryLoading,
-      value: fetchProjectsSummaryFailure
-        ? 'N/A'
-        : isEmpty(projectSummary)
-        ? '-'
-        : projectSummary.models_count
-    },
-    featureSets: {
-      className: 'default',
-      label: 'Feature sets',
-      loading: projectsSummaryLoading,
-      value: fetchProjectsSummaryFailure
-        ? 'N/A'
-        : isEmpty(projectSummary)
-        ? '-'
-        : projectSummary.feature_sets_count
-    },
-    files: {
-      className: 'default',
-      label: 'Files',
-      loading: projectsSummaryLoading,
-      value: fetchProjectsSummaryFailure
-        ? 'N/A'
-        : isEmpty(projectSummary)
-        ? '-'
-        : projectSummary.files_count
     }
+    // models: {
+    //   className: 'default',
+    //   label: 'Models',
+    //   loading: projectsSummaryLoading,
+    //   value: fetchProjectsSummaryFailure
+    //     ? 'N/A'
+    //     : isEmpty(projectSummary)
+    //     ? '-'
+    //     : projectSummary.models_count
+    // },
+    // featureSets: {
+    //   className: 'default',
+    //   label: 'Feature sets',
+    //   loading: projectsSummaryLoading,
+    //   value: fetchProjectsSummaryFailure
+    //     ? 'N/A'
+    //     : isEmpty(projectSummary)
+    //     ? '-'
+    //     : projectSummary.feature_sets_count
+    // },
+    // files: {
+    //   className: 'default',
+    //   label: 'Files',
+    //   loading: projectsSummaryLoading,
+    //   value: fetchProjectsSummaryFailure
+    //     ? 'N/A'
+    //     : isEmpty(projectSummary)
+    //     ? '-'
+    //     : projectSummary.files_count
+    // }
```

Comment:
I believe we still need this data. You probably commented it out because we don't have it on the design. You should ask someone about that

Reply:
I saw it is only being used in the projectCard.js. I commented it out in case of future usage. 
I'll check